impeller/renderer/renderer_unittests.cc

```diff
@@ -335,5 +338,70 @@ TEST_P(RendererTest, CanRenderInstanced) {
 }
 #endif  // IMPELLER_ENABLE_METAL
 
+#if IMPELLER_ENABLE_METAL  // The shader fails to link in the GLES backend for
+                           // some reason.
+TEST_P(RendererTest, TheImpeller) {
+  using VS = ImpellerVertexShader;
```

Comment:
There are two steps to take to disable the OpenGLES backend. The first is to make sure the test skips compilation (if you want it to of course). The second is to GTEST_SKIP the test with a message when the test is compiled and needs to be skipped at runtime. See the GTEST_SKIP pattern used for disabling the instancing test above. This is also why there is a presubmit failure.

Reply:
Done.